cores/outrun/cfg/macros.def

```diff
@@ -66,6 +66,14 @@ JTFRAME_CREDITS
 JTFRAME_CREDITS_PAGES=3
 JTFRAME_CREDITS_HSTART=32
 
+[demist_zx3a35]
+-JTFRAME_SKIP
+JTFRAME_LF_BUFFER
+JTFRAME_OSD_NOLOGO
```

Comment:
No BRAM even for the logo?

Reply:
I'll check if it has. I suppose yes, but at first try I disabled everything.